### DISCUSSION TITLE
tasklist underline

### DISCUSSION THREAD

**Discussion by istipisti113** [awesomeWM/awesome#3837] [Q&A] [answered]
I want to make so that the focused window on the taskbar on the wibar has an underline rather than change its background. I tried to search it on the web and ask chat-gpt but I couldn't manage to do so. I currently use the defaults in terms of the wibar.

**Comment by jeffmhubbard** [comment, marked as answer]
There's an example in the docs. [Stable version docs](https://awesomewm.org/doc/api/classes/awful.widget.tasklist.html) and [Git version docs](https://awesomewm.org/apidoc/widgets/awful.widget.tasklist.html)

**Reply by istipisti113** [reply]
thank you!
